File: week_1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

void pyramidrow(int n);

int main(void)
{
    int size = get_int("How big would you like your pyramid to be?\n");
    for (int i = 1; i < size+1; i++)
    {
        for (int k = 1; k <= size-i; k++)
        {
            printf(" ");
        }
        pyramidrow(i);
        printf("  ");
        pyramidrow(i);
        printf("\n");
    }
}

void pyramidrow(int n)
{
    for (int j = 1; j < n+1; j++)
    {
        printf("#");
    }
}

File: week_6/sentimental-mario-more/mario.py
from cs50 import get_int

size = 0
while size <= 0 or size > 8:
    size = get_int("Height: ")


for i in range(1, size+1):
    row = "#"*i
    print(" " * (size-i), end="")
    print(row ,"", row)
